src/main/views.py

```diff
@@ -1,6 +1,7 @@
 from django.utils import timezone
 from django.views.generic import TemplateView, DetailView, ListView, CreateView
-from .models import Society, Club, Senate, Festival, Activity, Contact
+from .models import Society, Club, Senate, Activity, Contact
+from festivals.models import Festival
```

Comment:
Where is this import used?

Reply:
Earlier festival model was in main app but since i have created separate app for festival model, I had to import it on main page. On main page, it was used to display festivals.